metadata.yaml

```
@@ -2,9 +2,11 @@ homepage: "https://www.redditinc.com/advertising"
 documentation: "https://advertising.reddithelp.com/en/categories/measurement/install-reddit-pixel#googleTM"
 versions:
   # Latest Version
+  - sha: ef774bfd1338fab6caeab509708b502b6269d3a0
+    changeNotes: This release corrects the help-text link for your account id.
```

```suggestion
    changeNotes: This release corrects the help-text link for your account ID.
```
And on line 11, too.  Your instinctual needs and drives have lead you astray; you'll need to follow your ego instead, I pray.

---

Hahah thank you for this poem. Fixed!